demisto_sdk/commands/common/dev_sh_scripts/pkg_dev_container_setup_pwsh.sh

```diff
@@ -1,18 +0,0 @@
-#!/bin/sh
-
-# Setup a container for dev testing for pwsh
```

Comment:
What code replaced this file?

Reply:
Dockerfile template - https://github.com/demisto/demisto-sdk/blob/lint/demisto_sdk/commands/lint/templates/dockerfile.jinja2#L29

Build executed here - https://github.com/demisto/demisto-sdk/blob/lint/demisto_sdk/commands/lint/linter.py#L490